content/running_bookwyrm/install-prod-dockerless.md

```diff
@@ -0,0 +1,147 @@
+---
+Title: Installing in Production
+Date: 2022-10-02
+Order: 2
+---
+
+This project is still young and isn't, at the moment, very stable, so please proceed with caution when running in production.
+This method of installation is more involved, and therefore is for more experienced admins. Docker install is recommended
+This install method assumes you already have ssl configured with certificates available
+
+## Server setup
+- Get a domain name and set up DNS for your server. You'll need to point the nameservers of your domain on your DNS provider to the server where you'll be hosting BookWyrm. Here are instructions for [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-point-to-digitalocean-nameservers-from-common-domain-registrars)
+- Set your server up with appropriate firewalls for running a web application (this instruction set is tested against Ubuntu 20.04). Here are instructions for [DigitalOcean](https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-20-04)
+- Set up an email service (such as [Mailgun](https://documentation.mailgun.com/en/latest/quickstart.html)) and the appropriate SMTP/DNS settings. Use the service's documentation for configuring your DNS
+- Install dependencies. On debian this could look like `apt install postgresql redis nginx python3-venv libpq-dev`
+
+## Install and configure BookWyrm
+
+The `production` branch of BookWyrm contains a number of tools not on the `main` branch that are suited for running in production, such as `docker-compose` changes to update the default commands or configuration of containers, and individual changes to container config to enable things like SSL or regular backups. Not all of these changes effect the dockerless install, however the `production` branch is still recommended
+
+Instructions for running BookWyrm in production:
+
+- Make and enter directory you want to install bookwyrm too. I suggest `/opt/bookwyrm`:
```

Comment:
For consistency, I'd like to avoid using first person. This could be phrased as "For example, `/opt/bookwyrm`:" or similar.